<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateTicketModal</class>
 <widget class="QWidget" name="UpdateTicketModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Билет</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="formLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="inputGroup_9">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="passengerLabel">
               <property name="text">
                <string>Пассажир</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="passengerComboBox"/>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QVBoxLayout" name="inputGroup_10">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="seatNoLabel">
               <property name="text">
                <string>Номер места</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="seatNoComboBox"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QVBoxLayout" name="inputGroup_11">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="railcarNoLabel">
               <property name="text">
                <string>Номер вагона</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="railCarNoComboBox"/>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="inputGroup_12">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="railcarClassLabel">
               <property name="text">
                <string>Класс обслуживания</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="railcarClassInput">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>2</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QVBoxLayout" name="inputGroup_7">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="scheduleLabel">
               <property name="text">
                <string>Рейс</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="scheduleComboBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Сохранить изменения</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
